src/components/cat-widget.tsx

```diff
@@ -105,10 +125,32 @@ export default function CatChatWidget({ propertyFeatures, className }: CatChatWi
                     >
                       {options ? (
                         <div className="grid grid-cols-2 gap-2">
-                          {options?.map((option) => (
-                            <Button key={option.text} variant="outline" onClick={() => handleAnswer(option)}>
-                              {option.text}
-                            </Button>
+                          {options?.map(({ text, action }) => (
+                            <motion.div
+                              className="w-full"
+                              key={text}
+                              initial="hidden"
+                              animate="visible"
+                              variants={{
+                                hidden: { opacity: 0, y: 20 },
+                                visible: (index) => ({
+                                  opacity: 1,
+                                  y: 0,
+                                  transition: {
+                                    delay: index * 0.2,
+                                    duration: 0.5,
+                                  },
+                                }),
+                              }}
+                            >
+                              <Button
+                                className="w-full"
+                                variant="outline"
+                                onClick={() => handleAnswer({ text, action })}
```

Comment:
Need to pass on whole option for the cat to reply with value rather than only text and action

Reply:
i think @nabing8 is working on that part

Reply:
This was done in a commit directly before this, might be rebased and mixed up